src/modelconfig.cpp

```diff
@@ -472,19 +472,23 @@ Status ModelConfig::parseModelMapping() {
         return StatusCode::JSON_INVALID;
     }
 
-    if (validateJsonAgainstSchema(doc, MODELS_MAPPING_INPUTS_SCHEMA) != StatusCode::OK) {
+    if (validateJsonAgainstSchema(doc, MODELS_MAPPING_SCHEMA) != StatusCode::OK) {
+        SPDLOG_ERROR("Configuration file is not a valid JSON file.");
```

Comment:
"Configuration file" is unclear if refers to config.json or mapping_config.json. Can we change to "mapping_config.json file is not valid JSON file"?

Reply:
Or state that mapping_config.json was found but it is invalid. Its optional to have mapping config file.

Reply:
We also change behavior, in case of failed mapping config parsing we stop model loading, previously we just warned an user. Is it intended?

Reply:
Agree with changing the error message.
As per behavior -> In L472 we can return error. This error is ignored anyway - and it shouldn't in model.cpp so for this we could change behavior and write test.